OpenLaTeXStudio/EditorModule/src/latexstudio/editor/remote/DisconnectDropbox.java

```diff
@@ -0,0 +1,73 @@
+/* 
+ * Copyright (c) 2016 Sebastian Brudzinski
+ * 
+ * See the file LICENSE for copying permission.
+ */
+package latexstudio.editor.remote;
+
+import com.dropbox.core.DbxClient;
+import com.dropbox.core.DbxException;
+import java.awt.event.ActionEvent;
+import java.awt.event.ActionListener;
+import javax.swing.JOptionPane;
+import latexstudio.editor.ApplicationLogger;
+import latexstudio.editor.DropboxRevisionsTopComponent;
+import latexstudio.editor.TopComponentFactory;
+import latexstudio.editor.settings.ApplicationSettings;
+import latexstudio.editor.settings.SettingsService;
+import org.openide.awt.ActionID;
+import org.openide.awt.ActionReference;
+import org.openide.awt.ActionRegistration;
+import org.openide.util.NbBundle.Messages;
+
+@ActionID(
+        category = "Remote",
+        id = "latexstudio.editor.remote.DisconnectDropbox"
+)
+@ActionRegistration(
+        displayName = "#CTL_DisconnectDropbox"
+)
+@ActionReference(path = "Menu/Remote", position = 3508)
```

Comment:
I think it would be nice to have the "Connect" and "Disconnect" actions grouped in the menu. Alternatively, the "Disconnect" could be the last option, as it is now, but with a separator between other options (similar to separator below "Connect")